### PR TITLE
Add sql server getting started guide

### PR DESCRIPTION
**Pull request by jamesongithub**
didn't have permissions to request a review. so just @ ing people

@bwgartner @chabowski @mfriesenegger @japelbaum

### CODE REVIEW

**Review by chabowski — APPROVED** [review]

**Review by chabowski — COMMENTED** [review]
Hi James, 
would you mind renaming the paper (and the DC file) to 
TRD-sql-server-getting-started.adoc
DC-TRD-sql-server-getting-started.adoc

This makes immediately clear that the document belongs to the Technical Reference Documents series.

File: adoc/sql-server-getting-started-docinfo.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:bugtracker>
+        <dm:url>https://github.com/SUSE/suse-best-practices/issues/new</dm:url>
+        <dm:product>SUSE Enterprise Linux Server</dm:product>
```
Comment (chabowski) [inline adoc/sql-server-getting-started-docinfo.xml:4]: The product name here should be the title of the paper ;-), in this case
SQL Server on SUSE Linux Enterprise: Getting Started

**Comment by jamesongithub** [comment]
> Hi James,
> would you mind renaming the paper (and the DC file) to
> TRD-sql-server-getting-started.adoc
> DC-TRD-sql-server-getting-started.adoc
> 
> This makes immediately clear that the document belongs to the Technical Reference Documents series.

updated, does it make sense to have a dir for each of them? trd, sles, cloud, storage, etc.?

**Comment by chabowski** [comment]
Hi James, 
I have merged your PR into the SBP main repo.

ATTENTION: branch master is now named main !!
Please update your local clone by running:

git branch -m master main
git fetch origin
git branch -u origin/main main

Thanks!
Meike

**Pull request merged**